draft-ietf-quic-http.md

```diff
@@ -552,10 +569,18 @@ an entire HTTP connection into a tunnel to a remote host. In HTTP/2, the CONNECT
 method is used to establish a tunnel over a single HTTP/2 stream to a remote
 host for similar purposes.
 
-A CONNECT request in HTTP/3 functions in the same manner as in HTTP/2. The
-request MUST be formatted as described in Section 8.3 of {{!HTTP2}}. A CONNECT
-request that does not conform to these restrictions is malformed (see
-{{malformed}}). The request stream MUST NOT be closed at the end of the request.
+A CONNECT request in HTTP/3 functions in the same manner as in HTTP/2.
```

Comment:
I think it would be simpler for a reader to assume everything functions in the same manner as HTTP/2, and only note when it doesn't.  Likewise, this document defines HTTP/3, so no need to repeat that either. IOW, an editorial suggestion I would go with:

  `A CONNECT request MUST be constructed as follows:`

just to make it much easier to read.

Actually, I would begin the paragraph with a one-line description of the method from http-core Semantics, with a normative reference, like

`The CONNECT method requests that the recipient establish a tunnel to the destination origin server identified by the request-target (Section X.x of [ref:Semantics]).  A CONNECT request MUST be constructed as follows:`